subsys/bluetooth/mesh/blob_io_flash.c

```diff
@@ -110,6 +110,12 @@ static int wr_chunk(const struct bt_mesh_blob_io *io,
 		    const struct bt_mesh_blob_chunk *chunk)
 {
 	struct bt_mesh_blob_io_flash *flash = FLASH_IO(io);
+
+#if defined CONFIG_SOC_FLASH_NRF_RRAM
```

Comment:
Could this be `IS_ENABLED(CONFIG_SOC_FLASH_NRF_RRAM)`? This will allow us to check that code at least compiles correctly without running a separate configuration in CI:
```
if (IS_ENABLED(CONFIG_SOC_FLASH_NRF_RRAM)) {
	return flash_area_write(flash->area,
				flash->offset + block->offset + chunk->offset,
				chunk->data, chunk->size);
}
```

Reply:
Done!